apps/expo/src/components/spotify/Spotify.tsx

```diff
@@ -0,0 +1,51 @@
+import { Dimensions, Image, Text, View } from 'react-native'
+import { StyleSheet } from 'react-native'
+
+import { GlobalContext } from '~/context/global-context'
+import { api } from '~/utils/api'
+
```

Comment:
That's good, install Prettier for vscode and format this file.

Now you have 2 options before i hit approve and merge in this pr

1. Have unit test for the spotfify component you wrote, its could be a bit tedious but I'll be on call with you

2. leave the unit tests on another beautiful day, just format it and push a new commits with the changes, I'll hit approve and merge for you from there 

lgtm